.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,53 +1,37 @@
-<!--
-Please use `pre-commit` to format code.
+1. Do not use AI, such as GPT, to help write your contribution.
+   The licensing implications are unclear.
 
-```
-pip install pre-commit  # install the package
-pre-commit install  # install git commit hook
-```
+2. Read our
+   [contribution guidelines](https://scikit-image.org/docs/dev/development/contribute.html)
+   if you have any questions.
 
-Now, formatting checks will be run on each commit.
-You can also run `pre-commit` manually:
+3. Use `pre-commit` to check and format code.
 
-```
-pre-commit run -a
-```
--->
+   Make it run automatically after each commit using:
+
+   ```
+   pip install pre-commit  # install the package
+   pre-commit install  # add git hook that runs checks after each commit
+   ```
+
+   Or, run it manually: `pre-commit run -a`
 
 ## Description
 
-<!-- If this is a bug-fix or enhancement, state the issue # it closes -->
+<!-- If this is a bug-fix or enhancement, provide the issue it closes. -->
```

Comment:
```suggestion
<!-- If this is a bug-fix or enhancement, provide the issue # it closes. -->
```
I feel this is more explicit. 
i.e provide the issue number not some other info like title, link or module.

Reply:
The hash gets read as a comment quite easily, so just make that `nr`?

Reply:
Thanks @decorouz. I incorporated your suggestion in a modified form in the trimmed down version.